tests/model_serving/constants.py

```diff
@@ -0,0 +1,2 @@
+KSERVE_SERVERLESS: str = "Serverless"
+KSERVE_RAWDEPLOYMENT: str = "RawDeployment"
```

Comment:
Just a nitpick, but maybe it would be a bit cleaner to have an enum, e.g.:

```
class KServeDeploymentType(Enum):
    SERVERLESS = "Serverless"
    RAW_DEPLOYMENT = "RawDeployment"
```